website/src/services/mapService.js

```diff
@@ -78,7 +78,7 @@ export function drawRouteOnMap(map, coordinates) {
  * @param {mapboxgl.Map} map The map on which to draw the marker.
  * @param {mapboxgl.LngLatLike} coordinates The coordinates of the marker.
  */
-export function drawMarkerOnMap(map, id, coordinates) {
+export function drawMarkerOnMap(map, id, coordinates, color) {
```

Comment:
Remember to update the above documentation to include the parameter. Also include a default fallback.